src/Fixer/Alias/PowToExponentiationFixer.php

```diff
@@ -0,0 +1,196 @@
+     *
+     * @return array[]
+     */
+    private function findPowCalls(Tokens $tokens)
+    {
+        $candidates = array();
+
+        // Minimal candidate to fix is seven tokens: pow(x,x);
+        $end = count($tokens) - 6;
+
+        // First possible location is after the open token: 1
+        for ($i = 1; $i < $end; ++$i) {
+            $candidate = $this->find('pow', $tokens, $i, $end);
+            if (null === $candidate) {
+                break;
+            }
+
+            $i = $candidate[1];
+
+            $arguments = $this->getArguments($tokens, $candidate[1], $candidate[2]);
+
+            if (2 !== count($arguments)) {
+                continue;
+            }
+
+            $candidate[] = $arguments;
+            $candidates[] = $candidate;
+        }
+
+        return $candidates;
+    }
+
+    /**
+     * @param Tokens         $tokens
+     * @param int            $functionNameIndex
+     * @param int            $openParenthesisIndex
+     * @param int            $closeParenthesisIndex
+     * @param array<int,int> $arguments
+     */
+    private function fixPowToExponentiation(Tokens $tokens, $functionNameIndex, $openParenthesisIndex, $closeParenthesisIndex, array $arguments)
+    {
+        // find the argument separator ',' directly after the last token of the first argument;
+        // replace it with T_POW '**'
+        $tokens->overrideAt(
+            $tokens->getNextTokenOfKind(reset($arguments), array(',')),
+            new Token(array(T_POW, '**'))
+        );
+
+        // clean up the function call tokens prt. I
+        $tokens[$closeParenthesisIndex]->clear();
+
+        // check if the arguments need to be wrapped in parenthesis
+        foreach (array_reverse($arguments, true) as $argumentStartIndex => $argumentEndIndex) {
+            if ($this->isParenthesisNeeded($tokens, $argumentStartIndex, $argumentEndIndex)) {
+                $tokens->insertAt($argumentEndIndex + 1, new Token(')'));
+                $tokens->insertAt($argumentStartIndex, new Token('('));
+            }
+        }
+
+        // clean up the function call tokens prt. II
+        $tokens[$openParenthesisIndex]->clear();
+        $tokens[$functionNameIndex]->clear();
+        $prev = $tokens->getPrevMeaningfulToken($functionNameIndex);
+        if ($tokens[$prev]->isGivenKind(T_NS_SEPARATOR)) {
+            $tokens[$prev]->clear();
+        }
+    }
+
+    /**
+     * @param Tokens $tokens
+     * @param int    $argumentStartIndex
+     * @param int    $argumentEndIndex
+     *
+     * @return bool
+     */
+    private function isParenthesisNeeded(Tokens $tokens, $argumentStartIndex, $argumentEndIndex)
+    {
+        static $allowedKinds;
+
+        if (null === $allowedKinds) {
+            $allowedKinds = array(T_DOUBLE_CAST, T_INT_CAST, T_CONSTANT_ENCAPSED_STRING, T_INC, T_DEC, T_DNUMBER, T_OBJECT_OPERATOR, T_LNUMBER, T_VARIABLE, T_STRING, T_NS_SEPARATOR, T_WHITESPACE, T_COMMENT, T_DOC_COMMENT, T_DOUBLE_COLON);
```

Comment:
it contains only compile-level known data, could be defined inplace (`śtatic $foo = [...]`)